[CS2103T-T10-2] DukeCooks

bigjunnn opened the pull request:
@hua-lun
@bakwxh
@khairulazman1997
@yyuanxin

meiannn commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,35 +296,36 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 [discrete]
-=== Use case: Delete person
+=== Use case: Add recipe
```
Consider numbering the use cases: e.g. Use case UC01: Add recipe

madanalogy commented on docs/DeveloperGuide.adoc:
```diff
@@ -279,13 +279,14 @@ Refer to the guide <<DevOps#, here>>.
+*Value proposition*: monitors all aspects of health in one application
 
 [appendix]
 == User Stories
```
Have you considered adding more user stories? Could consider including additional stories that would be possible in v2.0

meiannn commented on docs/DeveloperGuide.adoc:
```diff
@@ -337,17 +339,115 @@ Use case ends.
 * 3a. The given index is invalid.
```
Should this be generalised such that it doesn't show information about UI?

madanalogy commented on docs/DeveloperGuide.adoc:
```diff
@@ -337,17 +339,115 @@ Use case ends.
+Use case ends.
 
 [appendix]
 == Non Functional Requirements
```
Have you considered additional NFRs? For example, having backups in case of data corruption.

madanalogy commented on docs/DeveloperGuide.adoc:
```diff
@@ -337,17 +339,115 @@ Use case ends.
 
 .  Should work on any <<mainstream-os,mainstream OS>> as long as it has Java `11` or above installed.
-.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
+.  Should be able to hold up to 1000 recipes without a significant reduction in performance for typical usage.
```
Would you like to consider quantifying the "significant" reduction? This would help identifying whether the goal has been met.

meiannn commented on docs/DeveloperGuide.adoc:
```diff
@@ -337,17 +339,115 @@ Use case ends.
+
+[discrete]
+=== Use case: Add Calorie Intake
+
```
Maybe consider clarifying the information provided in Nutrition (Breakdown)? Its slightly unclear what the list of Nutrition means

meiannn commented on docs/DeveloperGuide.adoc:
```diff
@@ -337,17 +339,115 @@ Use case ends.
++
+Use case ends.
+
+*Extensions*
```
All your extensions are pretty exhaustive, good job!

ooimingsheng commented on docs/DeveloperGuide.adoc:
```diff
@@ -153,94 +181,369 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
+The class diagram below gives an overview of the `Recipe` class.
+
+.Recipe Class Diagram
+image::RecipeClassDiagram.png[]
```
Hi would it be better if the UML diagram has only either associations via the arrows or associations via the attributes?

ooimingsheng commented on docs/DeveloperGuide.adoc:
```diff
@@ -98,15 +110,15 @@ image::LogicClassDiagram.png[]
-.Interactions Inside the Logic Component for the `delete 1` Command
+.Interactions Inside the Logic Component for the `delete recipe 1` Command
 image::DeleteSequenceDiagram.png[]
 
```
Would it be better if the sequence diagram had bigger font to allow the reader to see more easily?

ooimingsheng commented on docs/DeveloperGuide.adoc:
```diff
@@ -153,94 +181,369 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
-The following sequence diagram shows how the undo operation works:
+Step 2. The user executes `add task tn/bake a cake td/12/12/2019` command to add a new
+task into Duke Cooks. The `add` command calls `Model#addDashboard()`, causing the task to
```
might want to consider splitting this into smaller steps because it looks quite cluttered to the reader in my opinion

hellodommy commented on docs/DeveloperGuide.adoc:
```diff
@@ -153,94 +181,369 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
+
+The following sequence diagram shows how the add operation works:
+
+image::AddTaskSequenceDiagram.png[]
```
Inconsistent that the style of the diagram changed.

hellodommy commented on docs/DeveloperGuide.adoc:
```diff
@@ -153,94 +181,369 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
-The following activity diagram summarizes what happens when a user executes a new command:
+The Exercise class is represented by the following class diagram below.
+
+image::ExerciseClassDiagram.png[]
```
Should choose to show association using attribute or with arrow, but not both.
There are also too many details in this diagram. Private attributions don't have to be shown. 🙈

hellodommy reviewed (commented):
Good effort on the diagrams! 👍🏻

bakwxh committed:
Merge pull request #159 from bakwxh/master

Create ViewMealPlan functionality

yyuanxin committed:
Add more test cases for health records

yyuanxin committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

* 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main:
  Write tests for MealPlan
  Write tests to improve coverage
  Fix EOF issue
  Fix UI for ViewMealPlan
  Add viewing of MealPlan
  Add EditPageCommand and corresponding tests
  Add styling to flowpane in DiaryCard
  Add more diary test cases
  Fix checkstyleTest
  Fix test cases
  Amend checkstyle
  Ammend result display for Recipe commands
  Make MealPlan creation and addition require existing Recipe
  Add more sample data to diary
  Add minor UI changes
  Add basic intro to PPP
  Write some StorageManager tests

# Conflicts:
#	src/test/java/dukecooks/logic/commands/CommandTestUtil.java

yyuanxin committed:
Merge pull request #160 from yyuanxin/master

Add more test cases and checks to health records

bakwxh committed:
Clean up tests for MealPlan Model components

bakwxh committed:
Add more tests to Recipe Model component

hua-lun committed:
Rename ppp

hua-lun committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main into update-ppp3

hua-lun committed:
Merge pull request #162 from hua-lun/update-ppp3

Update dev guide

bakwxh committed:
Write tests for various MealPlan and Recipe components

bakwxh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

 Conflicts:
	src/test/java/dukecooks/storage/StorageManagerTest.java

bakwxh committed:
Solve issues with import order

bakwxh committed:
Merge pull request #163 from bakwxh/master

Adding test cases

khairulazman1997 committed:
Adds view command functionality

khairulazman1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

 Conflicts:
	src/main/java/dukecooks/logic/parser/ViewCommandParser.java
	src/main/java/dukecooks/ui/MainWindow.java

yyuanxin committed:
Set to full screen window and fix sample loading error

yyuanxin committed:
Merge pull request #164 from yyuanxin/master

Set to full screen window and fix sample data loading error

khairulazman1997 committed:
Merge changes

khairulazman1997 committed:
Merge pull request #165 from khairulazman1997/master

Add view workout functionality

hua-lun committed:
Add icons for dashboard card

hua-lun committed:
Change font size

hua-lun committed:
Add credit for icons used

hua-lun committed:
Add more sample data

hua-lun committed:
Change date font colour

hua-lun committed:
Add test case for UniqueDashboardList

hua-lun committed:
Fix checkstyle errors

hua-lun committed:
Change diary to person

hua-lun committed:
Merge pull request #166 from hua-lun/update-ppp3

Add test cases and icons

bakwxh committed:
Modify FindMealPlanWith functionality

bakwxh committed:
Add images to UG

bakwxh committed:
Fix UG image issues

hua-lun committed:
Add goto commands

hua-lun committed:
Add DoneCommandParser

hua-lun committed:
Add DoneCommand

hua-lun committed:
Update ppp filename

hua-lun committed:
Remove DashboardCommandTest

hua-lun committed:
Merge pull request #167 from hua-lun/goto-function

Goto function

bigjunnn committed:
Fix edit diary command and delete diary command

bigjunnn committed:
Add limit to page title input and add page index to page card

bigjunnn committed:
Add diary name limit

bigjunnn committed:
Fix find diary command bug

bigjunnn committed:
Refactor create and add page

bigjunnn committed:
Add more sample diaries

bigjunnn committed:
Amend Javadocs and error messages for diary feature

bigjunnn committed:
Add auto-routing to diary tab for diary commands

khairulazman1997 committed:
Complete view exercise functionality

bigjunnn committed:
Add additional test cases

bigjunnn committed:
Fix checkstyle errors

bigjunnn committed:
Fix broken tests

bigjunnn committed:
Merge pull request #168 from bigjunnn/master

Update diary command and test cases

bigjunnn committed:
Add diary overview image to UG

bigjunnn committed:
Test images in UG

yyuanxin committed:
Fix profile and health records data in-sync issue

yyuanxin committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

* 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main:
  Fix broken tests
  Fix checkstyle errors
  Add additional test cases
  Add auto-routing to diary tab for diary commands
  Amend Javadocs and error messages for diary feature
  Add more sample diaries
  Refactor create and add page
  Fix find diary command bug
  Add diary name limit
  Add limit to page title input and add page index to page card
  Fix edit diary command and delete diary command

bakwxh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

hua-lun committed:
Add more test cases

hua-lun committed:
Merge pull request #169 from hua-lun/test-cases

Add more test cases

hua-lun committed:
Add redirect function to dashboard

hua-lun committed:
Update dashboard sample data

khairulazman1997 committed:
Adds new test files for exercise details

khairulazman1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

 Conflicts:
	src/main/java/dukecooks/ui/MainWindow.java

hua-lun committed:
Update dashboard display

khairulazman1997 committed:
Create test cases for Exercise Components

yyuanxin committed:
Add more test cases to profile and health

bigjunnn committed:
Edit UG

bigjunnn committed:
Fix EOF error

bigjunnn committed:
Fix EOF

bigjunnn committed:
Merge pull request #170 from bigjunnn/master

Update UG

bigjunnn committed:
Fix broken images

yyuanxin committed:
Add ClearRecordCommand, make UI changes to health records

yyuanxin committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

* 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main:
  Fix EOF
  Fix EOF error
  Edit UG
  Add more test cases
  Test images in UG
  Add diary overview image to UG

bigjunnn committed:
Merge pull request #171 from bigjunnn/master

Fix broken images in UG

bigjunnn committed:
Fix image issue in UG

bigjunnn committed:
Merge pull request #173 from bigjunnn/master

Fix image issue in UG

yyuanxin committed:
Fix EOF error
Set health records' type to be uneditable (i.e. Changing record type of existing records is disallowed.)

yyuanxin committed:
Merge pull request #172 from yyuanxin/master

Add test cases and edit Health Record's UI

bigjunnn committed:
Test theme for UG

bigjunnn committed:
Update fedora theme for UG

bigjunnn committed:
Fix EOF error

bigjunnn committed:
Merge pull request #174 from bigjunnn/master

Update UG theme

bigjunnn committed:
Test theme for UG

bigjunnn committed:
Merge pull request #175 from bigjunnn/master

Test theme for UG

khairulazman1997 committed:
Adds more tests for Exercises

khairulazman1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

khairulazman1997 committed:
Adds some javadoc comments

hua-lun committed:
Update UG and DG

khairulazman1997 committed:
Fix checkstyle errors

hua-lun committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main into update-ppp4

 Conflicts:
	docs/UserGuide.adoc

hua-lun committed:
Add line for EOF

hua-lun committed:
Merge pull request #177 from hua-lun/update-ppp4

Update ppp4

khairulazman1997 committed:
Fix checkstyle errors

khairulazman1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

khairulazman1997 committed:
Merge pull request #176 from khairulazman1997/master

Add more test cases and view exercise functionality

bakwxh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

bakwxh committed:
Add limit on Ingredient, Recipe, MealPlan names

yyuanxin committed:
Fix minor display issues

yyuanxin committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

* 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main:
  Fix checkstyle errors
  Add line for EOF
  Fix checkstyle errors
  Update UG and DG
  Adds some javadoc comments
  Adds more tests for Exercises
  Test theme for UG
  Fix EOF error
  Update fedora theme for UG
  Test theme for UG
  Fix image issue in UG
  Fix broken images
  Create test cases for Exercise Components
  Update dashboard display
  Adds new test files for exercise details
  Update dashboard sample data
  Add redirect function to dashboard
  Complete view exercise functionality

khairulazman1997 committed:
Adds exercise component to UG

khairulazman1997 committed:
Merge pull request #178 from khairulazman1997/master

Adds exercise component to UG

yyuanxin committed:
Update UG on health records section

yyuanxin committed:
Merge pull request #179 from yyuanxin/master

Update UG

yyuanxin committed:
Add higher resolution images to UG for health records section

yyuanxin committed:
Merge pull request #180 from yyuanxin/master

Update images in UG for health records

bigjunnn committed:
Update DG

bigjunnn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

 Conflicts:
	docs/DeveloperGuide.adoc

bigjunnn committed:
Add styling to documentation

bigjunnn committed:
Fix styling errors

bigjunnn committed:
Merge pull request #181 from bigjunnn/master

Update DG and styling to documentations

bakwxh committed:
Fix issues with GotoCommand, fix images for UG

bakwxh committed:
Flesh out UG more

bakwxh committed:
Add some 2.0 features that were removed previously

hua-lun committed:
Add exception when name entered is > 50

hua-lun committed:
Update ppp

hua-lun committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main into update-ppp4

hua-lun committed:
Update ppp

bakwxh committed:
Flesh out DG

bakwxh committed:
Flesh out command summary in UG

hua-lun committed:
Update ppp

bakwxh committed:
Adapt a draft PPP

bakwxh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

 Conflicts:
	docs/DeveloperGuide.adoc

bakwxh committed:
Merge pull request #182 from bakwxh/master

Some initial UG/DG/PPP

hua-lun committed:
Update ppp

hua-lun committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main into update-ppp4

yyuanxin committed:
Update DG for health records section

yyuanxin committed:
Add diagrams for DG

khairulazman1997 committed:
Updates the Workout Planner segment of the Developer's Guide

khairulazman1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

hua-lun committed:
Add limit characters to dashboardName

yyuanxin committed:
Update UG and DG for health section

hua-lun committed:
Update error messages

bigjunnn committed:
Update PPP

bigjunnn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

bigjunnn committed:
Merge pull request #184 from bigjunnn/master

Update DG and PPP

hua-lun committed:
Update ppp

bakwxh committed:
Update result display prompts

hua-lun committed:
Update target user statement

hua-lun committed:
Merge pull request #183 from hua-lun/update-ppp4

Update ppp4

bigjunnn committed:
Prepare PPP for submission

bigjunnn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

bakwxh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

hua-lun committed:
Update command format

khairulazman1997 committed:
Complete PPP and Developer Guide

khairulazman1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

khairulazman1997 committed:
Merge pull request #185 from khairulazman1997/master

Update Documentation

khairulazman1997 committed:
Updates AboutUs page

khairulazman1997 committed:
Merge pull request #186 from khairulazman1997/master

Updates AboutUs page

khairulazman1997 committed:
Fix doc images

khairulazman1997 committed:
Merge pull request #187 from khairulazman1997/master

Fix doc images

bigjunnn committed:
Minor edits to UG and update image figures

bigjunnn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

khairulazman1997 committed:
Fix checkstyle errors

yyuanxin committed:
Update about us

khairulazman1997 committed:
Merge pull request #188 from khairulazman1997/master

Fix checkstyle errors

bigjunnn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

yyuanxin committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

* 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main: (24 commits)
  Fix doc images
  Updates AboutUs page
  Complete PPP and Developer Guide
  Update target user statement
  Update ppp
  Update PPP
  Update error messages
  Add limit characters to dashboardName
  Updates the Workout Planner segment of the Developer's Guide
  Update ppp
  Adapt a draft PPP
  Update ppp
  Flesh out command summary in UG
  Flesh out DG
  Update ppp
  Update ppp
  Add exception when name entered is > 50
  Add some 2.0 features that were removed previously
  Flesh out UG more
  Fix issues with GotoCommand, fix images for UG
  ...

# Conflicts:
#	docs/DeveloperGuide.adoc

bigjunnn committed:
Merge pull request #190 from bigjunnn/master

Minor updates to UG and add figure images

bakwxh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

hua-lun committed:
Add logo

hua-lun committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main into update-ppp4

khairulazman1997 committed:
Updates user guide

khairulazman1997 committed:
Minor tweaks in doc

yyuanxin committed:
Fix EOF and minor changes to DG

bakwxh committed:
Add tables to DevGuide, polish PPP

yyuanxin committed:
Merge pull request #189 from yyuanxin/master

Update UG, DG, About Us

bakwxh committed:
Update AboutUs

bakwxh committed:
Fix issues with PPP

hua-lun committed:
Update dg

hua-lun committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main into update-ppp4

bakwxh committed:
Label images in PPP

hua-lun committed:
Update dg

bakwxh committed:
Amend EOF Checkstyle

khairulazman1997 committed:
Reduce PPP page count

bakwxh committed:
Amend some checkstyleMain issue

khairulazman1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

bakwxh committed:
Merge pull request #191 from bakwxh/master

Probably finale UG/DG/PPP changes

bigjunnn committed:
Minor formatting for DG

bigjunnn committed:
Fix image for header

bigjunnn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

khairulazman1997 committed:
Added new Test Cases

khairulazman1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

yyuanxin committed:
Add more sample data for health records

khairulazman1997 committed:
Merge pull request #194 from khairulazman1997/master

Reduce PPP page count

hua-lun committed:
Add manual testing

bakwxh committed:
Shorten PPP

khairulazman1997 committed:
Reduces pages of PPP

khairulazman1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

bakwxh committed:
Merge pull request #196 from bakwxh/master

Shorten PPP

yyuanxin committed:
update about us

khairulazman1997 committed:
Merge pull request #197 from khairulazman1997/master

Edits to PPP

bigjunnn committed:
Minor updates to UG

bigjunnn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

hua-lun committed:
Update about us

bigjunnn committed:
Update image figures in UG

hua-lun committed:
Update ppp

hua-lun committed:
Update command summary

khairulazman1997 committed:
Reduce PPP Pages

khairulazman1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

hua-lun committed:
Merge pull request #192 from hua-lun/update-ppp4

Update DG, about us

yyuanxin committed:
reduce sample

khairulazman1997 committed:
Merge pull request #199 from khairulazman1997/master

Edits PPP

bigjunnn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

 Conflicts:
	docs/DeveloperGuide.adoc
	docs/UserGuide.adoc

bakwxh committed:
Expand on manual testing

bigjunnn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

bakwxh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

 Conflicts:
	docs/DeveloperGuide.adoc

bigjunnn committed:
Merge pull request #198 from bigjunnn/master

Fix image issue in README and minor updates to UG

yyuanxin committed:
Revert "Add more sample data for health records"

This reverts commit da3b3758dc1b032e767bed8e674841088cb258a1.

yyuanxin committed:
Merge branch 'master' of https://github.com/yyuanxin/main

* 'master' of https://github.com/yyuanxin/main:
  reduce sample

# Conflicts:
#	src/main/java/dukecooks/model/util/SampleRecordDataUtil.java

bakwxh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

yyuanxin committed:
Increase coveralls

bakwxh committed:
Testing issues with scaling

bakwxh committed:
Truncate PPP

bigjunnn committed:
Fix images in UG

bigjunnn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

yyuanxin committed:
Fix coveralls issues

bakwxh committed:
Merge pull request #200 from bakwxh/master

Add more manual testing entries

bigjunnn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

yyuanxin committed:
Fix coveralls issues

yyuanxin committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

* 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main: (32 commits)
  Truncate PPP
  Testing issues with scaling
  Expand on manual testing
  Reduce PPP Pages
  Update command summary
  Update ppp
  Update image figures in UG
  Update about us
  Minor updates to UG
  Reduces pages of PPP
  Shorten PPP
  Add manual testing
  Added new Test Cases
  Fix image for header
  Minor formatting for DG
  Amend some checkstyleMain issue
  Reduce PPP page count
  Amend EOF Checkstyle
  Update dg
  Label images in PPP
  ...

yyuanxin committed:
Update User Guide and PPP

bigjunnn committed:
Final updates to UG / DG

yyuanxin committed:
Update PPP

yyuanxin committed:
Merge pull request #201 from yyuanxin/master

Update About Us

khairulazman1997 committed:
Edits on documentation

bigjunnn committed:
Update command messages

bigjunnn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

khairulazman1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

bigjunnn committed:
Merge pull request #202 from bigjunnn/master

Final updates to UG and DG

bigjunnn committed:
Update README.doc

hua-lun committed:
Update about us

yyuanxin committed:
Update PPP

bigjunnn committed:
Update figure numbers in UG

hua-lun committed:
Update figure number

hua-lun committed:
Merge pull request #203 from hua-lun/update-ppp4

Update figure number

bigjunnn committed:
Merge pull request #205 from bigjunnn/master

Update figure numbers in UG

yyuanxin committed:
Merge pull request #204 from yyuanxin/master

Update PPP

khairulazman1997 committed:
Adds new workouts to sample data

khairulazman1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T10-2/main

bakwxh committed:
Add sample data

bakwxh committed:
Merge pull request #207 from bakwxh/master

Add sample data

khairulazman1997 committed:
Merge pull request #206 from khairulazman1997/master

Adds new data to Sample Data

bigjunnn committed:
Rename chuahualun.adoc to hua-lun.adoc

bigjunnn committed:
Rename liewjunwei.adoc to bigjunnn.adoc

bigjunnn committed:
Rename leewah.adoc to bakwxh.adoc

bigjunnn committed:
Rename muhammadkhairulazman.adoc to khairulazman1997.adoc

bigjunnn committed:
Rename yeoyuanxin.adoc to yyuanxin.adoc

hua-lun committed:
Rename ppp

hua-lun committed:
Merge pull request #209 from hua-lun/update-ppp4

Rename ppp

hua-lun committed:
Update AboutUs.adoc

bigjunnn committed:
Update AboutUs.adoc

bigjunnn committed:
Update AboutUs.adoc
